lib/specifications/ci-types/jenkins/results.js

```diff
@@ -0,0 +1,83 @@
+/**
+ *
+ */
+'use strict';
+
+var XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest;
+var appConfig = require('../../../configuration/app-config').get();
+
+module.exports = {
+  getResults: getResults
+};
+
+
+/**
+ * Get or update the project repo using Git.
+ *
+ * @return a promise for the repo metadata.
```

Comment:
Are these docs accurate? The called function doesn't return anything and this function doesn't return anything. If you want to know when all the promises are resolved it is possible to operate on arrays of promises or a promise for an array of results.